bower.json

```diff
@@ -22,7 +22,14 @@
     "angular-sanitize": "^1.5.5",
     "firebase": "^2.4.2",
     "angularfire": "^1.2.0",
-    "isotope": "1.5.21",
-    "imagesloaded": "^4.1.0"
+    "imagesloaded": "^4.1.0",
+    "isotope": "^1.5.21"
+  },
+  "overrides": {
+    "font-awesome": {
+      "main": [
+        "css/font-awesome.css"
+      ]
+    }
   }
```

Comment:
You should be able to inject a file despite its version - and you can actually specify which version use.
I was aware it was an old version ...but I don't remember more why :P .....I will have a look and let u know!